src/editors/CobbDouglasEditor.jsx

```diff
@@ -33,15 +32,7 @@ export default class CobbDouglasEditor extends React.Component {
                             onChange={handleFormUpdate.bind(this)}
                         />
                     )}
-                    <MathJax.Context
```

Comment:
Does this component need to be replaced?

Reply:
oh yeah - thx

Reply:
Oh yeah - thx